web/src/settings_org.js

```diff
@@ -784,6 +784,7 @@ function set_up_dropdown_widget(setting_name, setting_options, setting_type) {
         on_mount_callback(dropdown) {
             if (setting_type === "group") {
                 $(dropdown.popper).css("min-width", "300px");
+                $(dropdown.popper).find(".simplebar-content").css("width", "max-content");
```

Comment:
Setting the width for `simplebar-content` element here because that's the first parent div for `ul` element and we do same for the popovers like gear menu, etc.